superset/assets/src/explore/components/AdhocFilterEditPopover.jsx

```diff
@@ -78,6 +79,13 @@ export default class AdhocFilterEditPopover extends React.Component {
     document.removeEventListener('mousemove', this.onMouseMove);
   }
 
+  adjustHeight(heightDifference) {
+    this.setState(state => ({
+      ...state,
```

Comment:
Do you need this? I thought you could just return the update to state.

Reply:
good call fixing now